MvvmCross.Forms/Platforms/Android/Views/MvxFormsSplashScreenActivity.cs

```diff
@@ -3,14 +3,15 @@
 // See the LICENSE file in the project root for more information.
 
 using MvvmCross.Core;
+using MvvmCross.Droid.Support.V7.AppCompat;
 using MvvmCross.Forms.Platforms.Android.Core;
 using MvvmCross.Platforms.Android.Views;
 using MvvmCross.ViewModels;
 using Xamarin.Forms;
 
 namespace MvvmCross.Forms.Platforms.Android.Views
 {
-    public abstract class MvxFormsSplashScreenActivity<TMvxAndroidSetup, TApplication, TFormsApplication> : MvxSplashScreenActivity
+    public abstract class MvxFormsSplashScreenActivity<TMvxAndroidSetup, TApplication, TFormsApplication> : MvxSplashScreenAppCompatActivity
```

Comment:
I guess we need to split this up into 2 splashscreens because it would be possible to build a Forms app without AppCompat.

Reply:
We could also say, AppCompat or bust.

Reply:
@martijn00 @Cheesebaron decision on whether to do a version without appcompat?

Reply:
Yeah let's go for AppCompat. It's the future!